hack/import_docs.sh

```diff
@@ -44,22 +51,35 @@ popd
 
 rm -rf $WORKDIR
 
-if [ ! -f content/docs/$release_version.md ]
-then
-  hugo new docs/$release_version.md
-  git add content/docs/$release_version.md
-fi
+# Detect the current version (one with the highest release number that is not RC)
+latest_version=$(ls $DOCDIR | sort -n | grep -v '\-rc\?' | tail -1)
+LATEST_VERSION_LINK_DIR=current
 
-# Detect the current version (one with the highest release number)
-current_version=$(ls $DOCDIR | sort -n | tail -1)
-if [ -d $DOCDIR/current ]
+# Preview release
+if [ $PREVIEW_RELEASE -eq 1 ]
 then
-    git rm -fr $DOCDIR/current
+  cat > $TARGETDIR/robots.txt <<EOF
+User-agent: *
+Disallow: /
+EOF
```

Comment:
The created file will be visible as `/documentation/1.24.0rc1/robots.txt`.
AFAIR it's not supported to have a `robot.txt` file in a directory different from the root.